src/Bicep.Core.IntegrationTests/ScenarioTests.cs

```diff
@@ -957,5 +957,134 @@ public void Test_Issue1432()
                 });
             }
         }
+
+        [TestMethod]
+        public void Test_Issue1028_1()
+        {
+            var (template, diags, _) = CompilationHelper.Compile(
+                ("main.bicep", @"
+resource kv 'Microsoft.KeyVault/vaults@2019-09-01' existing = {
+  name: 'testkeyvault'
+}
+
+
+module secret 'secret.bicep' = {
+  name: 'secret'
+  params: {
+    mySecret: kv.getSecret('mySecret')
```

Comment:
Would you mind adding an example of this syntax to the baselines at `/src/Bicep.Core.Samples/Files/Modules_CRLF` - just as we often use them to understand changes to the syntax structure, and I think you're the first user of object methods 😄

Reply:
In fact, some negative tests under `/src/Bicep.Core.Samples/Files/InvalidModules_LF` would probably also be valuable, as they also help to visually understand what the diagnostics will look like.

Reply:
I added some negative tests and the warnings didn't pop up in all places where they should... will be fixing this.